src/models/dto/paymentGetRequest.ts

```diff
@@ -0,0 +1,6 @@
+export default interface PaymentGetRequest {
```

Comment:
This should be PaymentPostRequest - we are not calling the Get endpoint from the frontend